docs/integrations/sources/mysql.md

```diff
@@ -180,6 +180,7 @@ If you do not see a type in this list, assume that it is coerced into a string.
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.4.9 | 2021-11-02 | [7559](https://github.com/airbytehq/airbyte/pull/7559) | Fixed Value '65535' is outside valid range for type |
```

Comment:
```suggestion
| 0.4.9 | 2021-11-02 | [7559](https://github.com/airbytehq/airbyte/pull/7559) | Correctly process large unsigned short integer values which may fall outside java's `Short` data type capability|
```

Reply:
Looks good, thanks for the suggestion. Commited.